scripts/utils.ts

```diff
@@ -124,17 +124,38 @@ export async function setupApis(
  *  Ive spent hours trying to mock download
  *
  * @param name - Api name to search for
- * @param deployment - What deployment to build for
+ * @param deployment - What deployment to build for (optional)
  * @param rootPath - Root path to download to
  *
  * @returns a promise that we will complete
  */
+export async function updateApis(name: string, rootPath: string): Promise<void>;
```

Comment:
It may be easier to just add new function with a different name (e.g. `downloadLatestApis`) and mark the old function deprecated. Also the code will be cleaner/simpler

Reply:
Thanks @unandyala I agree. The code is cleaner/simpler.
Done. I deprecated `updateApis` and added the new `downloadLatestApis` function.